source/client/index.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-import { render } from 'react-dom';
-import createApp from 'shared/components/app';
+import { renderToString } from 'react-dom/server';
+import { RoutingContext } from 'react-router';
 
-const App = createApp(React);
-const props = window.payload || {
-  title: 'Default client title'
+import { Provider } from 'react-redux';
+import configureStore from 'shared/configureStore';
+
+const store = configureStore(window.BOOTSTRAP_CLIENT_STATE);
+
+store.subscribe(() => console.log(store.getState()));
+
+const createDOM = renderProps => {
+  return renderToString(
+    <Provider store={store}>
+      <RoutingContext { ...renderProps } />
+    </Provider>
+  );
 };
 
-render(
-  <App { ...props } />,
-  document.getElementById('root')
-);
+export default createDOM;
```

Comment:
Ha @eb88 thanks. You've pointed me in the right direction. I've fixed the issue. There was indeed something wrong with the client-side rendering. Reducers are being updated now. Only issue left is the fact that Webpack doesn't want to parse the `app-home.js` file in the root directory. Really strange, this worked before. Not a webpack expert.

It gives me the following error:
```
You may need an appropriate loader to handle this file type.
| export default {
|   APP_HOME: __dirname
| };
```